pkgs/applications/science/math/lp_solve/default.nix

```diff
@@ -1,40 +1,45 @@
-{ lib, stdenv, fetchurl }:
+{ lib, stdenv, fetchurl, cctools, fixDarwinDylibNames }:
 
 stdenv.mkDerivation rec {
 
   pname = "lp_solve";
```

Comment:
```suggestion
stdenv.mkDerivation rec {
  pname = "lp_solve";
```

Reply:
Ah, but it was already merged. I think it's ok to leave there for now until a future update?